versioned_hdf5/tests/test_api.py

```diff
@@ -577,6 +578,64 @@ def test_resize_unaligned():
 
         file.close()
 
+def test_resize_multiple_dimensions():
+    # Test semantics against raw HDF5
+
+    shapes = range(5, 25, 5) # 5, 10, 15, 20
+    chunks = (10, 10, 10)
+    with setup() as f:
+        file = VersionedHDF5File(f)
+        for i, (oldshape, newshape) in\
+            enumerate(itertools.combinations_with_replacement(itertools.product(shapes, repeat=3), 2)):
+            print(oldshape, newshape)
```

Comment:
Is this print supposed to stay in the test?

Reply:
Oops, no.

Reply:
Although it would be cool if we could parameterize this test, similar to the InMemoryArrayDataset test in test_wrappers.py. The test takes 2 minutes to run and it would be nice to see the progress of it in the test runner output. The problem is that we have to reuse the same file for all the parameters. Recreating a file for each iteration makes the test take 10 times longer (https://github.com/deshaw/versioned-hdf5/pull/72#issuecomment-658452371). I don't know if pytest fixtures support reusing a tempfile fixture across parameterizations.